handbook/engineering/web/goals.md

```diff
@@ -4,29 +4,44 @@
 
 ### Long term
 
-**_Deliver the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users._**
+**_The web team has four areas of focus:_**
 
-**Outcome**: Our webapp, browser extensions and native integrations are great platforms to provide our users unique value through extensions. These platforms are well-maintained, consistent, easy to setup, well-documented, well-tested, performant, and show their power through convincing extensions built on top of them. The extensions platform and API provide powerful capabilities to extension developers and a great developer experience.
+  1. **_Creating and maintaining a highly usable and intentionally designed webapp interface_**
+  1. **_Delivering the full, unique value of Sourcegraph [extensions](https://docs.sourcegraph.com/extensions)_**
+  1. **_Maintaining and expanding code host [integrations](https://docs.sourcegraph.com/integration)_**
```

Comment:
```suggestion
  1. **_Maintaining and expanding [code host integrations](https://docs.sourcegraph.com/dev/background-information/web/code_host_integrations)_**
```

We own front-end code host integrations, but not back-end integrations (the actual connections to code host respositories). Would this doc page be more accurate?

Reply:
Good call! I'd looked at both of these and had originally wanted to shy away from a /dev/ page (was looking for an overview page) but I think you're right that the /dev/ page is more accurate. For now, I say let's go for it, and I'll note to update this if/when we get a better overview page into the docs.